test/unit_overapproximate.jl

```diff
@@ -1,3 +1,5 @@
+using Optim
```

Comment:
This is not needed (we have this command in `runtests.jl` already). Your test does not work with `Float32`, so you have to move this block of tests out of the loop (into the last loop).

Reply:
Thank you!